packages/react-core/src/components/DescriptionList/DescriptionListTerm.tsx

```diff
@@ -5,16 +5,20 @@ import { css } from '@patternfly/react-styles';
 export interface DescriptionListTermProps extends React.HTMLProps<HTMLElement> {
   /** Anything that can be rendered inside of list term */
   children: React.ReactNode;
+  /** icon that is rendered inside of list term */
```

Comment:
Nit.  Sentence case here please.

Reply:
I think it would be good to note where it is rendered with respect to the rest of the children.